internal/testclient/testeventsquery/client.go

```diff
@@ -0,0 +1,15 @@
+package testeventsquery
+
+import (
+	"testing"
+
+	"pocket/internal/testclient"
+	"pocket/pkg/client"
+	eventsquery "pocket/pkg/client/events_query"
+)
+
+func NewLocalnetClient(t *testing.T, opts ...client.Option) client.EventsQueryClient {
```

Comment:
Updated to `opts ...client.EventsQueryClientOption` in [de4defec](https://github.com/pokt-network/poktroll/pull/64/commits/de4defec5dd85d0182ba9cc1457bbc10f16a1e32#diff-b114b2cac95bfcb1700e8dcbc3de5a1d2e2d4cd7e74c4231d8c4f26a94feb89c)